Makefile

```diff
@@ -705,6 +705,7 @@ out/doc/api/assets/%: doc/api_assets/% out/doc/api/assets
 
```

Comment:
Failed with, not quite sure the location.
```bash
if [ ! -r node -o ! -L node ]; then ln -fs out/Release/node node; fi
make -s test-doc
added 101 packages in 1.192s
internal/modules/cjs/loader.js:612
    throw err;
    ^
Error: Cannot find module '/home/travis/build/nodejs/node/tools/doc/../node_modules/eslint/node_modules/js-yaml'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:610:15)
    at Function.Module._load (internal/modules/cjs/loader.js:539:25)
    at Module.require (internal/modules/cjs/loader.js:664:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/travis/build/nodejs/node/tools/doc/common.js:4:3)
    at Module._compile (internal/modules/cjs/loader.js:735:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:746:10)
    at Module.load (internal/modules/cjs/loader.js:627:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:570:12)
    at Function.Module._load (internal/modules/cjs/loader.js:562:3)
make[2]: *** [out/doc/api/addons.html] Error 1
make[1]: *** [doc-only] Error 2
make: *** [test] Error 2
The command "PARALLEL_ARGS='--flaky-tests=skip' make -j1 test" exited with 2.
```

Reply:
That's because of things like this in `tools/doc/common.js`:

```js
require(`${__dirname}/../node_modules/eslint/node_modules/js-yaml`);
```

That line was introduced in https://github.com/nodejs/node/pull/20701.

Reply:
Thanks.